hugo/config.yaml

```
@@ -29,8 +29,8 @@ params:
   github_repo: https://github.com/grafana/grafana-operator
   github_branch: master
   path_base_for_github_subdir:
-    from: content/(.*?)
-    to: docs/$1
+    from: ^.*?/grafana-operator/(?:docs/(examples/.*)|(docs/docs/.*)|(examples/.*))$
+    to: $1$2$3
```

Would something like this be sufficient? :)

```suggestion
    from: .*?/grafana-operator/(.*)
    to: $1
```